src/interfaces/jira.ts

```diff
@@ -115,12 +115,14 @@ export enum JiraCommitFileChangeTypeEnum {
 }
 
 export interface JiraAuthor {
-	avatar?: string;
 	email: string;
-	name: string;
+	name?: string;
+	avatar?: string;
 	url?: string;
 }
+
 export interface JiraReview extends JiraAuthor {
+	login: string;
```

Comment:
This shouldn't be in this type. `login` is not required by the API and the only place that uses it is for an implementation detail to reduce the users by login first before getting the extra information.

Instead, I'd recommend you add a custom type (extending the base type) nearing to the implementation that uses it.

Reply:
The type definition is still here and public, but isn't needed anywhere else but in the transform-pull-request file.  It also causes confusion between this and `JiraReview`